go.mod

```diff
@@ -219,7 +219,8 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.13.5 => github.com/ethereum-optimism/op-geth v1.101305.1-rc.1.0.20240103210259-c7871bc4454f
+//replace github.com/ethereum/go-ethereum v1.13.5 => github.com/ethereum-optimism/op-geth v1.101305.0-rc.2.0.20231220194825-94ca8ca83871
+replace github.com/ethereum/go-ethereum v1.13.5 => /Users/danyal/code/ext/op-geth
```

Comment:
Requires this to be merged: https://github.com/ethereum-optimism/op-geth/pull/214